ks/azure/centos76-hpc-sriov.ks

```diff
@@ -0,0 +1,401 @@
+cd ..
+cd && rm -rf /tmp/mpi
+
+# Setup module files for MPIs
+mkdir -p /usr/share/Modules/modulefiles/mpi/
+
+# HPC-X
+cat << EOF >> /usr/share/Modules/modulefiles/mpi/hpcx-v2.3.0
+#%Module 1.0
+#
+#  HPCx 2.3.0
+#
+conflict        mpi
+prepend-path    PATH            /opt/hpcx-v2.3.0-gcc-MLNX_OFED_LINUX-4.5-1.0.1.0-redhat7.6-x86_64/ompi/bin
+prepend-path    PATH            /opt/hpcx-v2.3.0-gcc-MLNX_OFED_LINUX-4.5-1.0.1.0-redhat7.6-x86_64
+prepend-path    LD_LIBRARY_PATH /opt/hpcx-v2.3.0-gcc-MLNX_OFED_LINUX-4.5-1.0.1.0-redhat7.6-x86_64/ompi/lib
+prepend-path    MANPATH         /opt/hpcx-v2.3.0-gcc-MLNX_OFED_LINUX-4.5-1.0.1.0-redhat7.6-x86_64/ompi/share/man
+setenv          MPI_BIN         /opt/hpcx-v2.3.0-gcc-MLNX_OFED_LINUX-4.5-1.0.1.0-redhat7.6-x86_64/ompi/bin
+setenv          MPI_INCLUDE     /opt/hpcx-v2.3.0-gcc-MLNX_OFED_LINUX-4.5-1.0.1.0-redhat7.6-x86_64/ompi/include
+setenv          MPI_LIB         /opt/hpcx-v2.3.0-gcc-MLNX_OFED_LINUX-4.5-1.0.1.0-redhat7.6-x86_64/ompi/lib
+setenv          MPI_MAN         /opt/hpcx-v2.3.0-gcc-MLNX_OFED_LINUX-4.5-1.0.1.0-redhat7.6-x86_64/ompi/share/man
+setenv          MPI_HOME        /opt/hpcx-v2.3.0-gcc-MLNX_OFED_LINUX-4.5-1.0.1.0-redhat7.6-x86_64/ompi
+EOF
+
+# MPICH
+cat << EOF >> /usr/share/Modules/modulefiles/mpi/mpich-3.3
+#%Module 1.0
+#
+#  MPICH 3.3
+#
+conflict        mpi
+prepend-path    PATH            /opt/mpich-3.3/bin
+prepend-path    LD_LIBRARY_PATH /opt/mpich-3.3/lib
+prepend-path    MANPATH         /opt/mpich-3.3/share/man
+setenv          MPI_BIN         /opt/mpich-3.3/bin
+setenv          MPI_INCLUDE     /opt/mpich-3.3/include
+setenv          MPI_LIB         /opt/mpich-3.3/lib
+setenv          MPI_MAN         /opt/mpich-3.3/share/man
+setenv          MPI_HOME        /opt/mpich-3.3
+EOF
+
+# MVAPICH2
+cat << EOF >> /usr/share/Modules/modulefiles/mpi/mvapich2-2.3
+#%Module 1.0
+#
+#  MVAPICH2 2.3
+#
+conflict        mpi
+prepend-path    PATH            /opt/mvapich2-2.3/bin
+prepend-path    LD_LIBRARY_PATH /opt/mvapich2-2.3/lib
+prepend-path    MANPATH         /opt/mvapich2-2.3/share/man
+setenv          MPI_BIN         /opt/mvapich2-2.3/bin
+setenv          MPI_INCLUDE     /opt/mvapich2-2.3/include
+setenv          MPI_LIB         /opt/mvapich2-2.3/lib
+setenv          MPI_MAN         /opt/mvapich2-2.3/share/man
+setenv          MPI_HOME        /opt/mvapich2-2.3
+EOF
+
+# OpenMPI
+cat << EOF >> /usr/share/Modules/modulefiles/mpi/openmpi-4.0.0
+#%Module 1.0
+#
+#  OpenMPI 4.0.0
+#
+conflict        mpi
+prepend-path    PATH            /opt/openmpi-4.0.0/bin
+prepend-path    LD_LIBRARY_PATH /opt/openmpi-4.0.0/lib
+prepend-path    MANPATH         /opt/openmpi-4.0.0/share/man
+setenv          MPI_BIN         /opt/openmpi-4.0.0/bin
+setenv          MPI_INCLUDE     /opt/openmpi-4.0.0/include
+setenv          MPI_LIB         /opt/openmpi-4.0.0/lib
+setenv          MPI_MAN         /opt/openmpi-4.0.0/share/man
+setenv          MPI_HOME        /opt/openmpi-4.0.0
+EOF
+
```

Comment:
Do we no longer need to install the Intel MPI?

Reply:
I added the popular open-source MPI's here. 

We can add Intel MPI (2019 update 2) as well. But I don't know if we have this internally. If you know about this, please feel free to add it.

Reply:
Or we can do this in later iterations.